.eslintrc.json

```diff
@@ -20,14 +21,19 @@
     "plugin:react-hooks/recommended"
   ],
   "rules": {
-    // Customized rules
-    "@typescript-eslint/no-extra-semi": "off", // Prettier work
+    "@typescript-eslint/no-extra-semi": "off",
     "@typescript-eslint/prefer-for-of": "error",
     "@typescript-eslint/unified-signatures": "error",
+    "@typescript-eslint/ban-ts-comment": "off",
+    "@typescript-eslint/no-explicit-any": "off",
 
     "import/no-deprecated": "error",
     "import/no-extraneous-dependencies": "error",
 
+    "react-hooks/exhaustive-deps": "off",
```

Comment:
We don't want to turn this off. This is a valuable check and bad things will slip in the code without it.